test/grpc_web_server.dart

```diff
@@ -164,4 +168,54 @@ if you are running tests locally.
     tempDir.deleteSync(recursive: true);
   }
   channel.sink.add('EXITED');
+
+  await server.shutdown().catchError((_) {});
+  await httpServer.close().catchError((_) {});
```

Comment:
Are these ignore-all-errors catches necessary here? Could they hide failures in the tests?

Reply:
Removed `catchError`.